tools/StaticAnalysis/ExampleAnalyzer/utils.ps1

```diff
@@ -469,38 +465,96 @@ function Get-ScriptAnalyzerResult {
         elseif($analysisResult.Severity -eq "Information"){
             $Severity = 3
         }
+        $locationMessage = $CodeMap[$analysisResult.Line - 1]
         if($analysisResult.RuleSuppressionID -ge 5000 -and $analysisResult.RuleSuppressionID -le 5199){
             $result = [AnalysisOutput]@{
-                Module = ($analysisResult.Message -split "-")[0]
-                Cmdlet = ($analysisResult.Message -split "-")[1] + "-" + ($analysisResult.Message -split "-")[2]
-                Example = ($analysisResult.Message -split "-")[3]
+                Module = $locationMessage.Module
+                Cmdlet = $locationMessage.Cmdlet
+                Example = $locationMessage.Example
+                Line = $locationMessage.Line
                 RuleName = $analysisResult.RuleName
-                Description = ($analysisResult.Message -split "#@#")[1] -replace "`"","`'" -replace "`n"," " -replace "`r"," "
+                Description = ($analysisResult.Message -split "#@#")[0] -replace "`"","`'" -replace "`n"," " -replace "`r"," "
                 Severity = $Severity
                 Extent = $analysisResult.Extent.ToString().Trim() -replace "`"","`'" -replace "`n"," " -replace "`r"," "
                 ProblemID = $analysisResult.RuleSuppressionID
-                Remediation = ($analysisResult.Message -split "#@#")[2] -replace "`"","`'" -replace "`n"," " -replace "`r"," "
+                Remediation = ($analysisResult.Message -split "#@#")[1] -replace "`"","`'" -replace "`n"," " -replace "`r"," "
             }
         }
         else{
             $result = [AnalysisOutput]@{
-                Module = ""
-                Cmdlet = ""
-                Example = 0
+                Module = $locationMessage.Module
+                Cmdlet = $locationMessage.Cmdlet
+                Example = $locationMessage.Example
+                Line = $locationMessage.Line
                 RuleName = $analysisResult.RuleName
                 Description = $analysisResult.Message -replace "`"","`'" -replace "`n"," " -replace "`r"," "
                 Severity = $Severity
                 Extent = $analysisResult.Extent.ToString().Trim() -replace "`"","`'" -replace "`n"," " -replace "`r"," "
                 ProblemID = 5200
-                Remediation = "Unexpected Error! Please check your example or contact the Azure Powershell Team. (Appeared in Line $($analysisResult.Line))"
+                Remediation = "Unexpected Error. Please check https://github.com/Azure/azure-powershell/blob/master/documentation/Debugging-StaticAnalysis-Errors.md for more details."
```

Comment:
```suggestion
                Remediation = "Unexpected Error. Please check [Trouble Shotting for Unexpected Errors in Example Issues](https://github.com/Azure/azure-powershell/blob/master/documentation/Debugging-StaticAnalysis-Errors.md#trouble-shotting-for-unexpected-errors-in-example-issues) for more details."
```
use hyperlink instead of url if you can